packages/sdk-core/previous-versions-testing/runQueryTests.ts

```diff
@@ -1,9 +1,7 @@
 import { Query } from "@superfluid-finance/sdk-core/src";
+import { chainIdToResolverDataMap } from "@superfluid-finance/sdk-core/src/constants";
```

Comment:
I don't see this import being used.

Reply:
it's used on line 13 by `getSubgraphEndpoint`